src/components/AppBar/index.jsx

```diff
@@ -29,16 +29,19 @@ const AppBar = () => {
         </Heading>
       </Box>
       <Box align="end" pad="xsmall">
-        {barButton(isLogged)}
+        {barButton(isLogged, userInfo)}
       </Box>
     </Grid>
   );
 };
 
-const barButton = (loggedin) => {
+const barButton = (loggedin, userInfo) => {
   const location = useLocation();
   if (loggedin) {
-    return <Box direction="row-responsive" gap="medium" justify="end"> <CreateOrder /> <Logout /> </Box>;
+    if (!userInfo.documentNumber) {
+      return <Box direction="row-responsive" gap="medium" justify="end"> <CreateOrder /> <Logout /> </Box>;
+    }
+    return <Box direction="row-responsive" gap="medium" justify="end"> <Logout /> </Box>;
   }
   if (!loggedin && location.pathname === '/login') {
     return <Box />;
```

Comment:
Se podria poner el condicional solo alrededor del componente createOrder, asi no se repite codigo. ALgo por el estilo se me ocurre
```suggestion
      return (
        <Box direction="row-responsive" gap="medium" justify="end"> 
        { !userInfo.documentNumber && 
          <CreateOrder />
        } 
        { loggedin && 
        <Logout />
        }
      </Box>)
```